src/ngraph/op/embeddingbag_offsets_sum.cpp

```diff
@@ -0,0 +1,183 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "ngraph/op/embeddingbag_offsets_sum.hpp"
+#include "ngraph/op/constant.hpp"
+
+using namespace std;
+using namespace ngraph;
+
+constexpr NodeTypeInfo op::v3::EmbeddingBagOffsetsSum::type_info;
+
+op::v3::EmbeddingBagOffsetsSum::EmbeddingBagOffsetsSum(const Output<Node>& emb_table,
+                                                       const Output<Node>& indices,
+                                                       const Output<Node>& offsets,
+                                                       const Output<Node>& per_sample_weights,
+                                                       const Output<Node>& default_index)
+    : Op({emb_table, indices, offsets, per_sample_weights, default_index})
+{
+    constructor_validate_and_infer_types();
+}
+
+op::v3::EmbeddingBagOffsetsSum::EmbeddingBagOffsetsSum(const Output<Node>& emb_table,
+                                                       const Output<Node>& indices,
+                                                       const Output<Node>& offsets,
+                                                       const Output<Node>& per_sample_weights)
+    : Op({emb_table, indices, offsets, per_sample_weights})
+{
+    constructor_validate_and_infer_types();
+}
+
+op::v3::EmbeddingBagOffsetsSum::EmbeddingBagOffsetsSum(const Output<Node>& emb_table,
+                                                       const Output<Node>& indices,
+                                                       const Output<Node>& offsets)
+    : Op({emb_table, indices, offsets})
+{
+    constructor_validate_and_infer_types();
+}
+
+void op::v3::EmbeddingBagOffsetsSum::validate_and_infer_types()
```

Comment:
Missing visitor

Reply:
visitor is only for attributes, isn't it?

Reply:
Correct. But even if the op does not have attributes we need to implement the method and return "true" so the whole system knows that the visitor is implemented and there are no attributes in the op.

@diyessi, please, correct me if I am wrong.